<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtDownloadManager</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/images/images/icon.png</normaloff>:/images/images/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="AddButton">
          <property name="text">
           <string>Add</string>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/images/images/buttons/AddButton.png</normaloff>:/images/images/buttons/AddButton.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="StartButton">
          <property name="text">
           <string>Start</string>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/images/images/buttons/StartButton.png</normaloff>:/images/images/buttons/StartButton.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="StopButton">
          <property name="text">
           <string>Stop</string>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/images/images/buttons/StopButton.png</normaloff>:/images/images/buttons/StopButton.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="EditButton">
          <property name="text">
           <string>Edit</string>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/images/images/buttons/EditButton.png</normaloff>:/images/images/buttons/EditButton.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="RevealButton">
          <property name="text">
           <string>Reveal</string>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/images/images/buttons/RevealButton.png</normaloff>:/images/images/buttons/RevealButton.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="RemoveButton">
          <property name="text">
           <string>Remove</string>
          </property>
          <property name="icon">
           <iconset resource="images.qrc">
            <normaloff>:/images/images/buttons/RemoveButton.png</normaloff>:/images/images/buttons/RemoveButton.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="SearchDownloadList">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeWidget" name="FiltersView">
       <column>
        <property name="text">
         <string>Filters</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>Downloads</string>
        </property>
        <property name="icon">
         <iconset resource="images.qrc">
          <normaloff>:/images/images/filterview/downloads.png</normaloff>:/images/images/filterview/downloads.png</iconset>
        </property>
        <item>
         <property name="text">
          <string>Completed</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/images/filterview/completed.png</normaloff>:/images/images/filterview/completed.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Running</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/images/filterview/running.png</normaloff>:/images/images/filterview/running.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Paused</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/images/filterview/paused.png</normaloff>:/images/images/filterview/paused.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Error</string>
         </property>
         <property name="icon">
          <iconset resource="images.qrc">
           <normaloff>:/images/images/filterview/error.png</normaloff>:/images/images/filterview/error.png</iconset>
         </property>
        </item>
       </item>
      </widget>
      <widget class="QTreeView" name="DownloadsList">
       <property name="minimumSize">
        <size>
         <width>550</width>
         <height>250</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="DownloadProgressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>745</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionReveal"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="actionEdit"/>
    <addaction name="actionRemove"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionNormal"/>
    <addaction name="actionFullScreen"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNormal">
   <property name="text">
    <string>Normal</string>
   </property>
  </action>
  <action name="actionFullScreen">
   <property name="text">
    <string>FullScreen</string>
   </property>
  </action>
  <action name="actionMinimize_to_tray">
   <property name="text">
    <string>Minimize to tray</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionReveal">
   <property name="text">
    <string>Reveal</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
